modules/build/src/main/scala/scala/build/preprocessing/directives/UsingScalaNativeOptionsDirectiveHandler.scala

```diff
@@ -33,7 +33,8 @@ case object UsingScalaNativeOptionsDirectiveHandler extends UsingDirectiveHandle
       "nativeGc",
       "nativeVersion",
       "nativeCompile",
-      "nativeLinking"
+      "nativeLinking",
+      "installClang"
```

Comment:
I'd rather use a key like `nativeClang`, and accept values such as `"system"` or `"managed"` for it (with the latter corresponding to the ones managed by the changes in this PR).